server/src/main/scala/org/http4s/server/staticcontent/MemoryCache.scala

```diff
@@ -2,8 +2,8 @@ package org.http4s
 package server
 package staticcontent
 
-import cats.effect._
-import cats.implicits._
+import cats.effect.Sync
+import cats.syntax.functor._
```

Comment:
I'd still prefer to use `cats.implicits._` here, but am open to being overruled.